aiohttp/web_reqrep.py

```diff
@@ -629,10 +634,22 @@ def start(coding):
                     return
 
     def start(self, request):
+        warnings.warn('use .prepare(request) instead', DeprecationWarning)
```

Comment:
Seems `.start` remain plain old function while `.prepare` is coroutine. Are they really interchangeable during deprecation period?